### PR TITLE
Issue #351

### PR DESCRIPTION
**Pull request by allaway**
Ok, so my source might be a bit controversial, but I figure it's just as good (if not better) than the "Sage Bionetworks" items. I couldn't find any of these terms in OLS4 and the spec sheets left me wanting something more pithy and informative.

### CODE REVIEW

File: modules/Data/FileFormat.yaml [hunk 1]
```diff
@@ -102,6 +102,15 @@ enums:
       cns:
         description: Segmented log2 ratios from CNVKit pipeline, these represent the most relevant output describing copy number variation. 
         source: https://cnvkit.readthedocs.io/en/stable/fileformats.html
```
Comment (allaway) [inline modules/Data/FileFormat.yaml:104]: @anngvu what is the difference between "meaning" and "source"? Should I be using one or the other?

Comment (anngvu) [reply modules/Data/FileFormat.yaml:104]: `meaning` should be an ontology uri. `source` is for some reference (so "Sage Bionetworks", maybe even "James Eddy", but most commonly some website or paper)

Comment (anngvu) [reply modules/Data/FileFormat.yaml:104]: I'll add this topic to the much-needed doc update...

Comment (allaway) [reply modules/Data/FileFormat.yaml:104]: Ok! I'll change this to "source" where relevant.

Comment (anngvu) [reply modules/Data/FileFormat.yaml:104]: So I don't think I completely segregated our original `source` values for terms into LinkML's particular of `meaning` vs `source` when I did the migration to LinkML, so this isn't too important for now. I think we can go back and clean this up when we do need to make sure that `meaning` are valid uris.

**Comment by anngvu** [comment]
Thanks, I think we can go back to move the `meaning` later to `source` since there are some terms where this needs to be refined as well.

**Review by anngvu — APPROVED** [review]

**Pull request merged**